Sources/StreamChatUI/Utils/ChatChannelNamer.swift

```diff
@@ -5,24 +5,28 @@
 import Foundation
 import StreamChat
 
-open class ChatChannelNamer {
-    open var maxMemberNames: Int { 2 }
-    open var separator: String { "," }
-    
-    public required init() {}
-    
-    /// Generates a name for the given channel, given the current user's id.
-    ///
-    /// The priority order is:
-    /// - Assigned name of the channel, if not empty
-    /// - If the channel is direct message (implicit cid):
-    ///   - Name generated from cached members of the channel
-    /// - Channel's id
-    /// - Parameters:
-    ///   - channel: Channel to generate name for.
-    ///   - currentUserId: Logged-in user. This parameter is used when deciding which member's names are going to be displayed.
-    /// - Returns: A valid Channel name.
-    open func name<ExtraData: ExtraDataTypes>(for channel: _ChatChannel<ExtraData>, as currentUserId: UserId?) -> String {
+public typealias ChatChannelNamer = _ChatChannelNamer
+public typealias _ChatChannelNamer<ExtraData: ExtraDataTypes> =
+    (_ channel: _ChatChannel<ExtraData>, _ currentUserId: UserId?) -> String
```

Comment:
These two typealiases should be documented, too. You should explain what a namer is.

Reply:
Thank you. Fixed.